focus-areas/ecosystem/social-currency-metric-system.md

```diff
@@ -1,4 +1,4 @@
-# Social Currency Metric System (SCMS)
+# Ecosystem Value: Social Currency Metric System (SCMS)
```

Comment:
We have been inconsistent with including the focus area in the title of a metric. Lately, we have removed them.

Can you please remove "Ecosystem Value: " from line 1?

Reply:
See below!